src/components/listView/ListView.js

```diff
@@ -1,39 +1,23 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import {ListViewInfo} from './ListView.view';
-
-import {FlatList} from 'react-native';
+import Axios from 'axios';
+import {Container} from '../container/Container';
 
 export const ListView = () => {
-  const tests = [
-    {
-      title: 'Test 1',
-      subtitle: 'Test one is about this',
-    },
-    {
-      title: 'Test 2',
-      subtitle: 'Test two is about this',
-    },
-    {
-      title: 'Test 3',
-      subtitle: 'Test three is about this',
-    },
-    {
-      title: 'Test 4',
-      subtitle: 'Test four is about this',
-    },
-    {
-      title: 'Test 5',
-      subtitle: 'Test five is about this',
-    },
-    {
-      title: 'Test 6',
-      subtitle: 'Test six is about this',
-    },
-    {
-      title: 'Test 7',
-      subtitle: 'Test seven is about this',
-    },
-  ];
 
-  return <ListViewInfo data={tests} />;
-};
+  const url = 'http://ec2-54-218-161-100.us-west-2.compute.amazonaws.com:3000/api/getalltests';
+  const [tests, setData] = useState();
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect( () => {
+    setIsLoading(true);
```

Comment:
i think you can remove line 13 if you set useState() which sets isLoading as a truthy value.